res/database/Default/database_programmes.xml

```diff
@@ -2399,7 +2449,7 @@ A bit more could have been made of such a confrontation, arguably.</en>
 				<member index="3" function="2">a5191c4e-7619-4aec-98ca-dde532991fca</member>
 			</staff>
 			<groups target_groups="0" pro_pressure_groups="0" contra_pressure_groups="0" />
-			<data country="USA" year="1979" distribution="1" maingenre="16" subgenre="" flags="0" blocks="3" price_mod="1" />
+			<data country="USA" year="1979" distribution="1" maingenre="16" subgenre="1,2" flags="8" blocks="3" price_mod="1" />
```

Comment:
I am not too happy about consindering every single of 10 movies as cult...

Reply:
With 10 you mean the Star Trek films? As there are "Trekkies", Star Trek conventions and all that, I think the flag is appropriate. Default SciFi results in the game by default are very bad anyway (and the Star Trek films indeed had actually only mediocre critics ratings and commercial success), so at least it boosts longevity and Teenage ratings a bit, which is the intended effect.

Reply:
I am happy enough to have mentioned that I am not too happy...